### DISCUSSION TITLE
Star Citizen Show Case

### DISCUSSION THREAD

**Discussion by Baconator650** [Markemp/Cryengine-Converter#82] [Show and tell]
Show off some of your progress using CGF-Converter.

![Airlock](https://user-images.githubusercontent.com/78010776/127327542-26d9cce6-bcc5-4ce8-92f4-7a777dccd72b.png)

![Front](https://user-images.githubusercontent.com/78010776/127327566-d41c4362-ec40-4623-850b-d9f0a6ea4fb7.png)

![Top](https://user-images.githubusercontent.com/78010776/127327592-9c119fc1-892b-481f-b934-c5f48a44b764.png)

![captainquarter](https://user-images.githubusercontent.com/78010776/127327634-b986ee5e-6f64-4c61-867c-109a91724737.png)

![hawk](https://user-images.githubusercontent.com/78010776/127327804-635e9e25-94c0-4b6d-8ad7-ed91a841e49c.png)

![Hurricane](https://user-images.githubusercontent.com/78010776/127327829-07fdc10b-278a-48e5-9e29-e191497512d1.png)

![MISCFL](https://user-images.githubusercontent.com/78010776/127327851-15ab660c-ada7-4627-a648-ff9f6353b3fc.png)

![Aegis_retaliator](https://user-images.githubusercontent.com/78010776/127327858-dbd58c30-a95f-43af-abf9-2ea43d5ac4f5.png)

![Tali_rear_bomb](https://user-images.githubusercontent.com/78010776/127327889-37d34643-ac53-41bf-8cb9-c153d6e39cca.png)

![Front](https://user-images.githubusercontent.com/78010776/127327913-328a15c3-34e4-46c4-be72-90bd867b26b8.png)

![back](https://user-images.githubusercontent.com/78010776/127327935-fa9cd2c9-3569-4fdf-affa-d94c1acaa050.png)

![Sent_int](https://user-images.githubusercontent.com/78010776/127327950-d2754034-93f3-4f3e-a520-2a1d1f557635.png)

![Harbinger](https://user-images.githubusercontent.com/78010776/127327967-d333ba31-1177-4c72-9b0a-222cb0365bde.png)

**Comment by vmxeo** [comment]
![ANVL_Pisces cutaway](https://user-images.githubusercontent.com/19572685/127344429-f65fb26d-716e-49bc-8110-e713a45483fe.png)

**Comment by Markemp** [comment]
It blows my mind that you guys can do this stuff.  It makes all the effort worth it.  Keep it up!

**Reply by Baconator650** [reply]
@Markemp you're very much part of this process. We wouldn't be able to do any of this if it weren't for you.

**Comment by vmxeo** [comment]
![Carrack cutaway](https://user-images.githubusercontent.com/19572685/127747432-e83b9ad9-ecf6-4e6f-bd65-428762da2643.png)

**Comment by MadochGreyson** [comment]
I've been putting these models into Blender for processing, which works in general.  The multiple parts (internal and external) aren't always aligned, and moving them around so far has been just me guessing about alignment positions.
Is there a reliable way to align these multiple objects for a single ship so everything aligns exactly?
Thanks for any advice you have....  -Madoch

**Reply by vmxeo** [reply]
There's another tool for this in the very early development stages. Stay tuned :)

**Reply by Markemp** [reply]
Oh man, that was going to be my reply!  Now that 1.3.1 is almost done, I was going to go back to Cryengine Importer and get SC prefabs all sorted out in there. :)

**Reply by MadochGreyson** [reply]
Fantastic! I'm glad to hear that.  My "guesses" haven't been accurate enough and the misalignments seem to cascade and amplify as I add more parts.  Thank you!

**Reply by Kjasi** [reply]
@MadochGreyson There's a tool called the [Star Citizen Data Viewer](https://gitlab.com/scmodding/tools/scdv) that can export Star Citizen ships into Blender with (virtually) everything included. (And it's being worked on all the time!) I highly recommend it, as their latest versions import ships that are very close to the in-game look, right out of the box!

That's where a lot of these new renders are coming from. (To be clear, SCDV just facilitates the conversion. Most of the heavy lifting is done by cgf-converter the Star Citizen texture converter. It DOES have a custom "Star Citizen Blueprint file" where it stores location information. This file is read by a SCDV plugin in Blender after the conversions and then loads everything up and into the correct locations.)

**Reply by MadochGreyson** [reply]
@Kjasi It sounds very intriguing. Do you know if there are any Star Citizen Data Viewer "walk-through" videos or directions out there?

**Reply by Kjasi** [reply]
@MadochGreyson It's not too well known, and very new (and changing all the time.) But it's pretty straightforwards. 

Quick run through: File->Open->Choose SC directory. Tools->Export Ship Entity, then select the ship you want to export, plus any settings. Export.

Then you'll want Tools->Blender->Install Blender Addon. Open Blender, File->Import->Import SCDT Blueprint, select the file you exported and you're done!

If you need more help, join their discord. I'm on there often.

**Reply by MadochGreyson** [reply]
@Kjasi I finally have some time to try out SCDV.  I saw the Sneak Peek that @Baconator650 demonstrated (with version 0.3.8), and it's exactly what I need. Unfortunately for me, the latest package that is available there is 0.3.1, which doesn't yet have the Blender integration.  It looks like the latest developer version is 0.3.10, but no packages available.
I'm a total noob when it comes to using Python, so I have no idea how to create an executable .msi package from the developer files.  I'll try to poke around and see if I can figure it out, but I thought I would ask if there's a plan to release a new package any time soon?
Thanks!

**Reply by Kjasi** [reply]
@MadochGreyson Try signing on to their Discord server. There's links to the packaged SCDV files there.

**Comment by Baconator650** [comment]
I'll be coming out with a video tutorial series on it, but right now it is currently in a heavy "Development Stage" so things are changing every week. Will most likely come out with a sneak peak of it once the next version gets release, to just quickly go over what SCDV does.

**Comment by Kjasi** [comment]
Using the new Vertex Colors in your materials:
![image](https://user-images.githubusercontent.com/5256123/128398514-0f6a2acc-0f8a-4e03-a1c2-8ccc3acc4e20.png)

**Comment by Baconator650** [comment]
![image](https://user-images.githubusercontent.com/78010776/128956157-32c4aeac-841f-421d-8e44-ea70071a5072.png)

![image](https://user-images.githubusercontent.com/78010776/128956167-e8d6c367-bddf-4f65-b381-68e54bce4890.png)

![image](https://user-images.githubusercontent.com/78010776/128956178-a7d96690-871e-468d-aca6-96c72d75155c.png)

**Comment by Baconator650** [comment]
![image](https://user-images.githubusercontent.com/78010776/129287030-8b3f264d-0e86-4f17-b04f-d65d04737b1b.png)

![image](https://user-images.githubusercontent.com/78010776/129287038-d89a6cc7-111a-4986-af9b-6eb5b7748754.png)

![image](https://user-images.githubusercontent.com/78010776/129287044-948bddcf-de5d-4956-bfd6-43f49bb3cb2f.png)

![image](https://user-images.githubusercontent.com/78010776/129431438-7b588b56-1ece-498a-8c13-4d2f77c9701a.png)

![image](https://user-images.githubusercontent.com/78010776/129431445-6e63559f-fb10-48c5-8d1e-ee18a842298a.png)

**Comment by Baconator650** [comment]
![](https://cdn.discordapp.com/attachments/426472147414941697/877232587788144650/unknown.png)


![](https://cdn.discordapp.com/attachments/426472147414941697/878491274984947792/unknown.png)


![](https://cdn.discordapp.com/attachments/426472147414941697/889252928085430323/unknown.png)

**Comment by vmxeo** [comment]
![Uploading Valkyrie Interior 01.png…](
![Star Runner Interior 01](https://user-images.githubusercontent.com/19572685/134669678-f4e85c46-ecd1-4463-a740-a8a0720ee3a8.png)
![Star Runner Interior 02](https://user-images.githubusercontent.com/19572685/134669688-97aa3914-7ec1-432b-b32b-cbfeb9310291.png)
![Catepillar Interior 01](https://user-images.githubusercontent.com/19572685/134669692-151967d1-f6b0-4416-986d-4e328e64a1da.png)
)

Just a few ship interiors :)

**Comment by vmxeo** [comment]
![Grim Hex 03](https://user-images.githubusercontent.com/19572685/134669887-aba14ee8-2692-4d79-a77b-16f00afcf749.png)
![Grim Hex 01](https://user-images.githubusercontent.com/19572685/134669891-0f328a64-fe21-4c13-a4e7-ae72a417d6ab.png)
![Levski Bar 02 (graded)](https://user-images.githubusercontent.com/19572685/134669930-8ef1d6a0-08b5-4954-bccb-783490303a80.png)
![Levski Bar 03](https://user-images.githubusercontent.com/19572685/134669939-095cf17f-8b39-4cbb-b3ef-8ed150918426.png)

**Comment by Baconator650** [comment]
![image](https://user-images.githubusercontent.com/78010776/162202224-1b043059-5522-425b-a346-a051bc441a4f.png)

**Comment by Baconator650** [comment]
![image](https://github.com/Markemp/Cryengine-Converter/assets/78010776/c9d9ec14-9a33-4d00-9175-f9c72454c0a8)

![image](https://github.com/Markemp/Cryengine-Converter/assets/78010776/a688d36d-cc4e-4529-b7d3-f5414ec347c4)

![image](https://github.com/Markemp/Cryengine-Converter/assets/78010776/467eec48-3368-409a-9b6c-22658215c759)

![image](https://github.com/Markemp/Cryengine-Converter/assets/78010776/08a30245-b949-4d9e-83bd-1e37b7551874)

![image](https://github.com/Markemp/Cryengine-Converter/assets/78010776/f8f5cd50-1d02-4a0c-b2ed-88f8d0d75ad5)

![image](https://github.com/Markemp/Cryengine-Converter/assets/78010776/52692908-9579-444c-970e-0ba15644b998)

**Comment by Baconator650** [comment]
![image](https://github.com/Markemp/Cryengine-Converter/assets/78010776/6f32cd82-2cec-4fe4-b935-fc39b2c48518)

![image](https://github.com/Markemp/Cryengine-Converter/assets/78010776/87676340-75c8-4976-9cc5-8421e7ea1a9d)

![image](https://github.com/Markemp/Cryengine-Converter/assets/78010776/cb89c552-4a7f-4b2d-b7b9-298c22132105)

**Reply by Markemp** [reply]
Damn those look good!

**Reply by Baconator650** [reply]
Thank You. Wouldn't have been possible without your work.

**Comment by Baconator650** [comment]
![image](https://github.com/Markemp/Cryengine-Converter/assets/78010776/726a1f19-4cc9-4827-be4a-7daf3c90cf31)

![image](https://github.com/Markemp/Cryengine-Converter/assets/78010776/5bd26b83-7e4f-4fe2-a488-18c8417b2023)
